benchmarks/Riok.Mapperly.Benchmarks/SourceGeneratorBenchmark.cs

```diff
@@ -0,0 +1,80 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Loggers;
+using Microsoft.Build.Locator;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.MSBuild;
+
+namespace Riok.Mapperly.Benchmarks;
+
+[MemoryDiagnoser]
+[InProcess]
+//[EventPipeProfiler(EventPipeProfile.CpuSampling)]
+//[EtwProfiler]
+//[DisassemblyDiagnoser]
```

Comment:
Shouldn't this commented code be removed before going to main? Or add comments explaining why it is commented and what activating these attributes does.